### ISSUE TITLE
[FEATURE REQUEST] - Sort by Size in Current Layer Contents

### ISSUE THREAD

**Issue by ondrovic** [wagoodman/dive#341]
Would be nice to sort asc / desc based on size   in the current layer contents

**Comment by whyman** [comment]
Yes, this would be amazing for easily finding the worst offenders!

**Comment by dragonfax** [comment]
Definitely would love this. Needed it today.

**Comment by iiian** [comment]
Unfortunately, it's been almost a year since this codebase has had any merge activity on it, and I suspect it may be essentially "done" at this point. If anybody wants this feature, you can grab it from https://github.com/iiian/dive. I'll add some build instructions to the readme, but it's incredibly simple, I was really excited when the thing just built, first-try, with literally zero fuss.

```
git clone git@github.com:iiian/dive.git
cd dive
make all
# your output will be in ./dist/dive_<your-target-triple>/dive
# for example, ./dist/dive_linux_amd64/dive
```

Then just grab it and toss it in your `usr/bin` or wherever.

Happy new year! 🥂🐳🤿

**Comment by evandrocoan** [comment]
Thanks @iiian , I was having trouble cloning it with ssh, but it worked with `git clone https://github.com/iiian/dive`

**Comment by jumarko** [comment]
@wagoodman is there any interest in incorporating this feature into the main repo?

**Comment by iiian** [comment]
@jumarko , I am pretty sure this has been merged into the main repo -- I have version 0.11.0 installed via `homebrew` & `^O` toggles sort.

We might be able to close this PR

**Comment by jumarko** [comment]
> `^O` toggles sort.

@iiian that's nice, I didn't know that.
It seems it's not documented, though - it would be nice to add that to the README.

**Comment by iiian** [comment]
Ah, sorry my tone might have been misleading. I'm just a contributor, I don't have repo control or anything, and it took quite awhile to get that PR merged in. Agree though, I probably should have made a documentation update as part of that PR

**Comment by sanmai-NL** [comment]
#89
